webapp/src/components/MortgageActions/MortgageActions.container.js

```diff
@@ -9,9 +16,18 @@ const mapState = (state, ownProps) => {
   }
 }
 
-const mapDispatch = (dispatch, { mortgage }) => ({
-  onCancel: () =>
-    dispatch(cancelMortgageRequest(mortgage.mortgage_id, mortgage.asset_id))
-})
+const mapDispatch = (dispatch, { mortgage }) => {
+  const coordinates = splitCoordinate(mortgage.asset_id)
```

Comment:
To be a bit more clear, we could change this to  `const [x, y] = splitCoordinate(asset_id)`

Also, looking at this I found a bug on `cancelMortgageSuccess` in `mortgage/actions.js`:

https://github.com/decentraland/marketplace/pull/244/files#diff-1bc520535984d0e30212dd309cfb59b1R46

```javascript
  const { x, y } = splitCoordinate(assetId)
// Should be
  const [ x, y ] = splitCoordinate(assetId)
```